adapters/Vungle/VungleAdapter/VungleRouterConfiguration.m

```diff
@@ -22,6 +22,14 @@ + (void)setPublishIDFV:(BOOL)publish {
   [VungleSDK setPublishIDFV:publish];
 }
 
++ (void)setCCPAStatus:(VungleCCPAStatus)ccpaStatus {
+  [[VungleSDK sharedSDK] updateCCPAStatus:ccpaStatus];
+}
+
++ (VungleCCPAStatus)getCCPAStatus {
```

Comment:
@YueVungle Can users access to `VungleCCPAStatus` without including `VungleSDK.h`? If not, we might need to add similar enum at adapter level, and adapter should convert it to `VungleCCPAStatus`. I can see we did wrong for "VungleConsentStatus" since it requires to include `VungleSDK.h`.

Reply:
@ashinagawa  Hi Aki, got it, I will add the similar enum in Adapter level. Thanks a lot for the reminder. ^_^

Reply:
@ashinagawa Hi Aki, updated, thanks a lot for the guidance. ^_^